src/main/java/com/bakdata/kafka/KafkaConnectorSourceResetter.java

```diff
@@ -124,6 +124,9 @@ private static Converter createConverter(final Map<String, Object> kafkaConfig)
 
     @Override
     public void run() {
+        if (this.offsetTopic.isBlank()) {
+            throw new IllegalArgumentException("--offset-topic should be set and cannot be blank.");
```

Comment:
I still think this is unneccessary. Picocli marks this arg as required and you implicitly refer to the argument name which can get messy if we change things in picocli config